libs/store/src/lib/infrastructure/persistence/store/entity-store.ts

```diff
@@ -37,13 +39,19 @@ export interface EntityStore<T> {
      * @param entity Entity to be deleted
      */
     delete(entity: T): Observable<T>;
+
+    /**
+     * Create new entity instance wrapped Proxy
+     * @param fromState initial state
+     */
+    createEntityInstance(fromState?: T): T;
```

Comment:
Actually `fromState` should be based on EntityDTO generic type, `T` in this context means entity class type, not a DTO type